src/arrayMethodFind.js

```diff
@@ -5,7 +5,13 @@
  */
 function applyCustomFind() {
   [].__proto__.find2 = function(callback) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      if (callback(this[i], i, this)) {
+        return this[i];
+      }
+    }
+
+    return undefined;
```

Comment:
Your function will return `undefined` anyway, so you can remove this line of code.